firmware/library/L3_HAL/sddriver.hpp

```diff
@@ -0,0 +1,53 @@
+#pragma once
+#include <cstdint>
+
+// Relevant Datasheet: https://www.sdcard.org/downloads/pls/pdf/index.php?p=Part1_Physical_Layer_Simplified_Specification_Ver6.00.jpg&f=Part1_Physical_Layer_Simplified_Specification_Ver6.00.pdf&e=EN_SS1
+class SdInterface {
+ public:
+    // TODO(#152): define a command frame struct
+    struct {
+        uint8_t cmd;
+        uint32_t arg;
+        uint8_t crc;
+    } commandFrame;
+
+    // TODO(#151): Define a response frame struct since there are various
+    // responses that will be sent, depending on the command sent previously
+    struct {
+        uint8_t response[6];
+    } responseFrame;
+
+    enum class SdCommand {
+        RESET       = 0x40 | 0,     // CMD0: reset sd card to idle state
```

Comment:
What does the 0x40 mean in this context and why do all of the commands include it?